SetGridField - Improved error message (#10911)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_3_12.md

#### Scripts
##### SetGridField

> - Improved error message for an incorrect grid ID. 

Reviewer: ```suggestion
- Improved the error message when a grid ID is incorrect. 
```
